tests/unit/wingsail/common/test_lut.py

```diff
@@ -0,0 +1,63 @@
+import math
+
+import numpy as np
+import pytest
+from scipy import interpolate
+
+from controller.common.lut import LUT
+
+
+class TestLUT:
+    # Intialize lookup table
+    look_up_table = [[50000, 5.75], [100000, 6.75], [200000, 7], [500000, 9.75], [1000000, 10]]
+
+    def test_LUT_constructor(self):
```

Comment:
What about testing the constructor with a numpy array rather than a list?

Reply:
tested both now:

```
    look_up_table = [[50000, 5.75], [100000, 6.75], [200000, 7], [500000, 9.75], [1000000, 10]]
    look_up_np_array = np.array(look_up_table)

    def test_LUT_constructor(self):
        # set up
        testLUT = LUT(self.look_up_table)
        testLUT2 = LUT(self.look_up_np_array)
        # test that LUT return a known value
        assert math.isclose(testLUT(50000), 5.75)
        assert math.isclose(testLUT2(50000), 5.75)
```